tests/test_colorinterp.py

```diff
@@ -13,13 +19,16 @@ def test_cmyk_interp(tmpdir):
     tiffname = str(tmpdir.join('foo.tif'))
     with rasterio.open(tiffname, 'w', **meta) as dst:
         assert dst.profile['photometric'] == 'cmyk'
-        assert dst.colorinterp(1) == ColorInterp.cyan
-        assert dst.colorinterp(2) == ColorInterp.magenta
-        assert dst.colorinterp(3) == ColorInterp.yellow
-        assert dst.colorinterp(4) == ColorInterp.black
+        dst.colorinterp = [
+            ColorInterp.cyan,
+            ColorInterp.magenta,
+            ColorInterp.yellow,
+            ColorInterp.black]
 
 
-@pytest.mark.skip(reason="crashing on OS X with Homebrew's GDAL")
+@pytest.mark.skipif(
+    os.environ.get('TRAVIS', os.environ.get('CI', 'false')).lower() != 'true',
+    reason="Crashing on OS X with Homebrew's GDAL")
```

Comment:
This might need to be revisited.  Worked fine for me with homebrew gdal 2.2.2

Reply:
Turns out this is still a problem, although I altered the `skipif()` so it runs on GDAL `2.2.2`, which also works for me.